sdk/appconfiguration/app-configuration/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,75 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-app-configuration
+urlFragment: app-configuration-javascript
+---
+
+# Azure App Configuration client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure App Configuration in some common scenarios.
+
+| **File Name**                                                   | **Description**                                                                                                                                                                                                                                                                                                                             |
+| --------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [getSettingOnlyIfChanged.js][getsettingonlyifchanged]           | Demonstrates getting a setting only if it has changed from what you already have. (This allows your app to avoid downloading the contents of a setting if the value is unchanged.)                                                                                                                                                          |
```

Comment:
This ordering seems a little weird. Maybe `helloworld` should be weighted first if it's meant to be an introduction to the API. If it's not meant to be an introduction to the API, maybe it should be renamed. Thoughts?

Reply:
Added weights.